Framework/Geometry/test/ComponentInfoTest.h

```diff
@@ -648,6 +650,33 @@ class ComponentInfoTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(infoScan1->position({0 /*detector index*/, 1}),
                      detectorPos);
   }
+
+  void test_StructuredPanel_for_single_rectangular_bank() {
+    auto instrument =
+        ComponentCreationHelper::createTestInstrumentRectangular2(1, 4);
+    auto wrappers = InstrumentVisitor::makeWrappers(*instrument);
+    const auto &componentInfo = std::get<0>(wrappers);
+
+    // find structured panel
+    size_t structuredIndex = 0;
```

Comment:
Minor point. Would it be so bad to hard-code the index as  `root - N`? I think that what you have is more flexible, but as this is test code, if the `cretateTestInstrumentRetangular` is changed, these tests would break and that would get fixed. Might save a few lines of code here as part of your search.